### ISSUE TITLE
[Feature]: Enhance Lightbox Viewer

### ISSUE THREAD

**Issue by RJSprod** [vladmandic/automatic#757]
### Feature description

The lightbox feature is difficult to use on mobile (IMO). Here are some suggested improvements.

--

**Make interactive area of arrows much much larger**
- The buttons are really small for a touch display.

**Allow taping image to go to next image** 
- Currently, tapping lightbox image closes the lightbox
- This is redundant behavior with the x button, would much rather tap go to next image

**Reduce header or only make it appear on a user action**
- Simply put, give the picture the option to full screen
- The actions buttons in the header can be collapsed until needed, maybe a button in the upper left "toggle options"

--

Screenshot: No option to disable lightbox
![Screenshot 2023-05-04 133250](https://user-images.githubusercontent.com/111059656/236284269-82beed4b-7394-4070-a93e-21ed598c5665.png)

**Comment by vladmandic** [comment]
And what is the use case for that?
Just because something existed doesn't mean it cannot removed. If you have a valid use case, please explain it.

**Comment by vladmandic** [comment]
Also, why write actual details in a closed item which is not what this is. What you wrote there is a good start, why is it not here. And I replied there.

**Comment by RJSprod** [comment]
> And what is the use case for that? Just because something existed doesn't mean it cannot removed. If you have a valid use case, please explain it.

On mobile, its just not very easy to tap the arrows. I want to quickly go through generations that i've made in a batch. There is not arrow keys on mobile, and the tiny arrows on the sides of the image are difficult to tap.

It would be one thing if tapping on the image performed "next image" but instead, it closes the light box. 

Additionally, the light box options take up space on the screen. 

I could suggest improvements, i'm not gradio dev myself.

1. Make interactive area of arrows much much larger (maybe the full height of the screen?)
2. Allow taping image to go to next image (why have tap close the lightbox when there is also a x button, redundancy)
3. Reduce header or only make it appear on a user action, like tap to reveal additional options

I know this is not your developed lightbox code, but these are the reason i disable it for my use cases.

If you don't want to make the option available i get it, still a great UI.

**Comment by vladmandic** [comment]
Everything you mention here is both sensible and viable, so please update actual feature request) or close it and use this writeup to creat new one, your choice).

**Comment by anapnoe** [comment]
we are working on the SD.Next release this enhancements have already been added to the future release you can try the UI here
https://github.com/anapnoe/stable-diffusion-webui-ux to see if it meets your standards
@vladmandic can you create a label SD.Next for the things that are going to be available in the official SD.Next release
